test/fixtures/minimal/components/sales/app/public/sales/errors.rb

```diff
@@ -0,0 +1,8 @@
+# typed: ignore
+# frozen_string_literal: true
+
+module Sales
+  module Errors
+    SomethingWentWrong = Class.new(RuntimeError)
+  end
```

Comment:
> I do think it could be even slimmer than it already is, and I welcome @Shopify/packwerk veterans' suggestions for what should be left out. After a quick settling period, minimal should not change very often.

Since this is the minimal skeleton do we need to have so many modules in it? Shouldn't it be enough if we keep it to `Orders`, `Sales::Orderes`, `RecordNewOrdere` to be able to build on top of it? WDYT?

Reply:
Those three make sense, and we can revisit once tests start using the fixture.